modules/restore-replace-stopped-baremetal-etcd-member.adoc

```diff
@@ -0,0 +1,612 @@
+openshift-control-plane-0 Ready master 4h26m v1.24.0+9546431
+openshift-control-plane-1 Ready master 4h26m v1.24.0+9546431
+openshift-control-plane-2 Ready master 12m   v1.24.0+9546431
+openshift-compute-0       Ready worker 3h58m v1.24.0+9546431
+openshift-compute-1       Ready worker 3h58m v1.24.0+9546431
+----
+
+.Verification
+
+. Verify that all etcd pods are running properly.
++
+In a terminal that has access to the cluster as a `cluster-admin` user, run the following command:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-etcd -o wide | grep etcd | grep -v guard
+----
++
+.Example output
+[source,terminal]
+----
+etcd-openshift-control-plane-0      5/5     Running     0     105m
+etcd-openshift-control-plane-1      5/5     Running     0     107m
+etcd-openshift-control-plane-2      5/5     Running     0     103m 
+----
++
+If the output from the previous command only lists two pods, you can manually force an etcd redeployment. In a terminal that has access to the cluster as a `cluster-admin` user, run the following command:
++
+[source,terminal]
+----
+$ oc patch etcd cluster -p='{"spec": {"forceRedeploymentReason": "recovery-'"$( date --rfc-3339=ns )"'"}}' --type=merge <1>
+----
++
+<1> The `forceRedeploymentReason` value must be unique, which is why a timestamp is appended.
++
+.To verify there are exactly three etcd members, connect to the running etcd container, passing in the name of a pod that was not on the affected node:
++
+.In a terminal that has access to the cluster as a `cluster-admin` user, run the following command:
++
+[source,terminal]
+----
+$ oc rsh -n openshift-etcd etcd-openshift-control-plane-0
+----
++
+. View the member list:
++
+[source,terminal]
+----
+sh-4.2# etcdctl member list -w table
+----
++
+.Example output
+[source,terminal]
+----
++------------------+---------+--------------------+---------------------------+---------------------------+-----------------+
+|        ID        | STATUS  |        NAME        |        PEER ADDRS         |       CLIENT ADDRS        |    IS LEARNER    |
++------------------+---------+--------------------+---------------------------+---------------------------+-----------------+
+| 7a8197040a5126c8 | started | openshift-control-plane-2 | https://192.168.10.11:2380 | https://192.168.10.11:2379 |   false |
+| 8d5abe9669a39192 | started | openshift-control-plane-1 | https://192.168.10.10:2380 | https://192.168.10.10:2379 |   false |
+| cc3830a72fc357f9 | started | openshift-control-plane-0 | https://192.168.10.9:2380 | https://192.168.10.9:2379 |     false |
++------------------+---------+--------------------+---------------------------+---------------------------+-----------------+
+----
++
+If the output from the previous command lists more than three etcd members, you must carefully remove the unwanted member.
++
+
+. Verify that all etcd members are healthy:
+
++
+[source,terminal]
+----
+# edctl endpoint health --cluster
```

Comment:
Did you mean `etcdctl` here?

Reply:
Fatigue. That's my only reasonable answer here. Just fatigue. DOH!